fileuploader-kiwiirc-plugin/components/fileListComponent.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="kiwi-filebuffer-outer-container">
+        <div v-if="typeof fileList === 'undefined' || !fileList.length" class="kiwi-filebuffer-status-message">
```

Comment:
fileList should never be undefined so that check isn't needed here

Reply:
Ah good call. That was a throwback to when the fileList array was in plugin.js, but yeah.. the check is not needed anymore.